### PR TITLE
Bitrise cloud signing guide

### PR DESCRIPTION
**Pull request by BucekJiri**

### CODE REVIEW

File: ci/bitrise-complete-guide-cloud-signing.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
+# Monstarlab iOS Bitrise - Setup Guide with Cloud Code Signing ☁️
+
+This is a simple Bitrise app setup guide that makes use of Apple cloud signing. This allows us to simplify the setup tremendously. 
+
+## How does cloud signing work? 
+Cloud signing means the code signing files are managed on Apple servers. You do not need to set them up locally or use Fastlane Match. The code signing files are fetched from Apple servers and you just need an Apple dev account with sufficient access rights (Admin or Developer with access to certificates).
```
Comment (mariusc) [inline ci/bitrise-complete-guide-cloud-signing.md]:
```suggestion
Cloud signing means the code signing files are managed on Apple servers. You do not need to set them up locally or use Fastlane Match. The code signing files are fetched from Apple servers and you just need an Apple dev account with sufficient access rights (Admin or Developer with access to certificates - including cloud certificates).
```

File: ci/bitrise-complete-guide-cloud-signing.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+
+## Code signing
+You only need to upload the development signing certificate to Bitrise. This is used to code sign an initial archive before the code signing files are downloaded from Apple and the app is resigned with them. Even this development certificate could be omitted theoretically because the cloud code signing can create a new one if it does not find it locally. This, however, creates multiple new development certificates with every new build so it is better to upload it to Bitrise.
+
+1) Download the certificate from the Apple Developer portal.
+<p align="center">
+  <img src="../images/ci-cloudSigning/5_1.png"/>
+</p>
+
+2) The certificate is in a .cer format. You will need the private key .p12 file though that is embedded inside the .cer file.
```
Comment (mariusc) [inline ci/bitrise-complete-guide-cloud-signing.md]:
```suggestion
2) The certificate is in a .cer format. You will need the private key .p12 file though that is associated with the .cer file.
```
The private key isn't embedded in the .cer file. Well, technically, the .cer files can contain private keys in them, but Apple's .cer files don't. They're just associated 😊

File: ci/bitrise-complete-guide-cloud-signing.md [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+3) Double click the certificate and install it into your Keychain.
+4) Find the cert in your Keychain. Expand it and right tap on the private key. Select ***Export <Certificate name>***. If you do not see the private key, you downloaded a certificate without it. You can either find the person who created the certificate and get the original certificate from them or you can create a new certificate. The full certificate with the private key can be downloaded from the Apple Developer portal only once.
```
Comment (mariusc) [inline ci/bitrise-complete-guide-cloud-signing.md]: > The full certificate with the private key can be downloaded from the Apple Developer portal only once.
Are you sure about this? I'm quite sure I was able to download the same certificate multiple times from the Apple Developer portal

**Review by mariusc — APPROVED** [review]
Looks good; I added a few suggestions, feel free to adopt them or not and merge when ready 👍

File: ci/bitrise-complete-guide-cloud-signing.md (hunk 3 above)
Comment (BucekJiri) [reply ci/bitrise-complete-guide-cloud-signing.md]: You are right. I am wrong. I thought the private key is downloaded with the .cer file only for the first time. Just tried it now and looks like the private key is put into the keychain when you create the Signing Request and when you download the .cer file, the private key is associated with it in the keychain. I will rewrite this part a bit.

File: ci/bitrise-complete-guide-cloud-signing.md (hunk 2 above)
Comment (BucekJiri) [reply ci/bitrise-complete-guide-cloud-signing.md]: Yes. Correct. Thank you 💪

**Pull request merged**